src/.nuget/packages.builds

```diff
@@ -29,9 +25,19 @@
     <Project Include="Microsoft.NETCore.ILDAsm\Microsoft.NETCore.ILDAsm.builds" />
   </ItemGroup>
 
-  <Import Project="$(ToolsDir)versioning.targets" />
-  <!-- Make sure we create version.txt file since it will be packaged -->
-  <Target Name="EnsureVersionInfoFileExists" BeforeTargets="Build" DependsOnTargets="CreateVersionInfoFile" />
+  <!-- Generate a version.txt file we include in our packages -->
+  <Target Name="GenerateVersionInfoFileForPackages"
+          BeforeTargets="Build"
+          DependsOnTargets="InitializeSourceControlInformationFromSourceControlManager">
```

Comment:
Can you comment on where or at least what system provides the `InitializeSourceControlInformationFromSourceControlManager` target?

Reply:
In fact, I'm skeptical of why we are doing this. The SDK + arcade do this, provided the right set of imports.

Reply:
Could you point at where / what target or task? I did not find one when I was looking. I only found one for generating a native header, not a simple text file with just the SourceRevisionID.

Reply:
I failed to see why you were doing this, and yes, this seems to be the easiest way to do it. Although I wonder, @RussKeldorph and @dagood, are these files really used anywhere? I'd expect the resource strings in coreclr/S.P.C to be enough unless we are using them for something else?